v1/linkml-schemas/credentials.yaml

```diff
@@ -31,16 +31,25 @@ classes:
   AuthorizationRequestClass:
     is_a: JWTClass
     slots:
-      - shared_claims
       - response_type
       - scope
       - client_id
       - state
       - redirect_uri
       - nonce
-      - prompt
+
+  RPAuthorizationRequestClass:
```

Comment:
We've named these two things inconsistently. Here, the name is for the client (ie the RP), but in the `CRIAuthorizationRequestClass` case, you've named it for the server (the CRI).

Perhaps we should look for consistency with OpenID Connect, for the RP case, perhaps call it [`OpenIDConnectAuthenticationRequest`](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).

In the CRI case, my starter for 10 would be `IssuerAuthorizationRequest` as I want to avoid "CRI" as an abbreviation wherever possible, because it causes confusion with "CI" for contra-indicator.

Reply:
Class names updated based on comments.